pkg/prober/prober.dockerfile

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
Don't add non-go code to `pkg/`. I'm thinking to have `prober` folder at the repo root.

Reply:
Done. Moved to repo root.